### ISSUE TITLE
RuntimeWarning: coroutine 'TurismoClient.run' was never awaited tc.run() 

### ISSUE THREAD

**Issue by vincent2v** [RaceCrewAI/gt-telem#3]
Hi, 
i really appreciate your work - thank you!
I encounter a problem when trying to start with your given simple expample.

I´m getting the Error: "RuntimeWarning: coroutine 'TurismoClient.run' was never awaited tc.run()"
![gt-telem](https://github.com/RaceCrewAI/gt-telem/assets/60935303/8cfbbf8e-21d3-4934-b72c-8e7d3c307ea5)

I googled quite a qhile, but unfortunatly i´m a noob :(

Can you help me solving the issue?
Do you need any further information?

Thank you in Advance

**Comment by Jonpro03** [comment]
Hey! Sorry, I need to update docs. They are out of date. 
`await tc.run()` should do the trick.
Please let me know

**Comment by Jonpro03** [comment]
Correction, for the example, it's this:
```python
import asyncio
import json

from gt_telem import TurismoClient
from gt_telem.models import Telemetry

async def print_telem(t: Telemetry):
    print(json.dumps(t.as_dict, indent=4))

tc = TurismoClient()
tc.register_callback(print_telem)
asyncio.run(tc.run())
```
I'll update the docs

**Comment by vincent2v** [comment]
Thank you for your fast reply!
That did the job! :)

Looking foorward for updatet documents / examples with events and maybe some explanation on how to layout my own printout / read specific values.

Thank you so much for this! ❤

**Comment by Jonpro03** [comment]
Resolved with 0.2.9 which adds start/stop, run and run_async methods to TurismoClient to support a variety of use-cases.
https://github.com/RaceCrewAI/gt-telem#usage

**Comment by vincent2v** [comment]
Thx m8!
